deps/build.jl

```diff
@@ -62,6 +62,12 @@ if is_windows()
     push!(BinDeps.defaults, BuildProcess)
 end
 
+# OSX
+if is_apple()
```

Comment:
Add
```julia
using Compat.Sys: iswindows, isapple, isunix
```
after the `using Compat` line and replace all `is_<os>` with `is<os>`.  You should also rise `Compat` version to 0.29.0 in `REQUIRE`.